Gopkg.lock

```diff
@@ -573,19 +579,22 @@
   revision = "master"
 
 [[projects]]
+  branch = "master"
   name = "github.com/xeipuuv/gojsonpointer"
   packages = ["."]
-  revision = "6fe8760cad3569743d51ddbb243b26f8456742dc"
+  revision = "4e3ac2762d5f479393488629ee9370b50873b3a6"
```

Comment:
Was this updated intended?

Reply:
I don't control what gets updated in the lock file.

Reply:
The changes I do are only in the toml file, this is autogenerated by dep.